src/components/core/PivotTable.tsx

```diff
@@ -153,14 +246,46 @@ export class PivotTableInner extends
         }
     }
 
+    /**
+     * getCellClass returns class for drillable cells. (maybe format in the future as well)
+     */
+    public getCellClass = (classList: string) => (cellClassParams: CellClassParams): string => {
+        const { drillableItems } = this.props;
+        // return none if no drillableItems are specified
+        if (drillableItems.length === 0) {
+            return null;
+        }
+
+        const afm: AFM.IAfm = this.props.dataSource.getAfm();
+
+        const { rowIndex } = cellClassParams;
+        const colDef = cellClassParams.colDef as IGridHeader;
+        const rowDrillItem =
+        get<CellClassParams, IDrillableItem>(cellClassParams, ['data', 'drillItemMap', colDef.field]);
+
+        const drillItems = rowDrillItem ? [...colDef.drillItems, rowDrillItem] : colDef.drillItems;
+        const hasDrillableHeader = drillItems
+            .some(
+                (drillItem: Execution.IResultHeaderItem) => isDrillable(drillableItems, drillItem, afm)
+            );
+
+        const className = classNames(
+            classList,
+            getCellClassNames(rowIndex, colDef.index, hasDrillableHeader),
+            colDef.index !== undefined ? `gd-column-index-${colDef.index}` : null,
+            colDef.measureIndex !== undefined ? `gd-column-measure-${colDef.measureIndex}` : null
+        );
+        return className;
+    }
+
     public createDataSource = (resultSpec: AFM.IResultSpec, getPage: IGetPage) => {
-        const onSuccess = (execution: Execution.IExecutionResponses, columnDefs: IGridHeader[]) => {
+        const onSuccess = (execution: Execution.IExecutionResponses, columnDefs: ColDef[]) => {
```

Comment:
so this is ColDef and obove is colDef as IGridHeader?

Reply:
replaced by IGridHeader